### DISCUSSION TITLE
7.8.8 在windows10下启动后崩溃

### DISCUSSION THREAD

**Discussion by turbin** [getlantern/lantern-forum-cn#715] [Q&A]
[lantern.log](https://github.com/user-attachments/files/16186421/lantern.log)

启动后崩溃。无法打开，目前最新版本。
蓝灯最近太不稳定了。

![image](https://github.com/user-attachments/assets/e608da93-1209-4829-ae7b-e0d5ff982db1)
系统信息。

**Comment by turbin** [comment]
是不是能开放历史版本下载？找一下之前可以用的版本。不然我都打算退订了。

**Reply by ycjdixon** [reply]
@turbin
之前的版本也很难联网了。

**Reply by trickyloler** [reply]
你可以下载win7那个版本的链接试下

**Reply by ycjdixon** [reply]
WIN7版本已经停止更新了，停留在7.8.2版本了。

**Reply by helanhao** [reply]
闪退的解决办法是：安装win7版本的就可以了，安装之后它会自动升级到7.8.4版本，不会闪退了

**Reply by sun12yyds** [reply]
重装Windows算了

**Comment by MaxMKMax** [comment]
各位好. 非常抱歉影响了您的体验感. 如果有谁还没提交报告的可以帮我们个忙. 您那里能帮我提交报告然后给我 8 票号.吗? 如果应用闪退, 提交不了, 您可以手动帮我找log然后提供给我吗? 非常感谢! 有log我们就能马上修复了. 麻烦您帮忙了. 谢谢! 
可以是看这么找:
C:\Users\username\AppData\Roaming\Lantern

各位放心. 我已经把你们的问题提交给技术团队了. 我们会尽快修复的. 谢谢您的耐心等待. 也很抱歉这几天影响了你们的体验感. 谢谢你们的反馈也感谢你们支持蓝灯!

**Comment by MaxMKMax** [comment]
@turbin 各位好. 可以帮我们提交一个应用内报告吗? 或者用手动方法给我们log. 谢谢. 明天开会我帮你们反馈问题给技术团队, 尽快修复. 麻烦你们啦. 谢谢帮忙.
